### PR TITLE
changed mousePressed reference description issue#6045 and variable name of example

### LINKED ISSUE #6045: Misleading description of mousePressed() reference
### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [X] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build Process
- [ ] Unit Testing
- [ ] Internalization
- [ ] Friendly Errors
- [ ] Other (specify if possible)

### p5.js version

v1.5.0

### Web browser and version

108.0.5359.125 (Official Build) (64-bit) (cohort: Stable)

### Operating System

Windows

### Steps:
1. go to the webpage [link](https://p5js.org/reference/#/p5/mousePressed)
2.  the description of the mouse Pressed i.e. Click within the image to change
 the value of the rectangle, is wrong as we don't really need to click within the rectange , clicking anywhere would still change the value. 
 also I think we should call the variable color_value instead of value only.

### Snippet:
image

```

### PR DESCRIPTION
**Pull request by yogitheboss**
Resolves #6045

 Changes:

 Screenshots of the change:
image

### CODE REVIEW

**Comment by yogitheboss** [comment]
@limzykenneth can you please review this pr

File: src/events/mouse.js [hunk 1]
```diff
@@ -611,20 +611,20 @@ p5.prototype._onmousemove = function(e) {
...
  *   rect(25, 25, 50, 50);
  *   describe('black 50-by-50 rect turns white with mouse click/press.');
  * }
  * function mousePressed() {
  *   if (value === 0) {
- *     value = 255;
+ *     color_value = 255;
  *   } else {
- *     value = 0;
+ *     color_value = 0;
```
Comment (limzykenneth) [inline src/events/mouse.js:627]: Instead of `color_value` can we use `colorValue`? Just to keep consistent code style.

**Comment by yogitheboss** [comment]
I have made the necessary changes @limzykenneth

**Comment by yogitheboss** [comment]
@limzykenneth  I have made the necessary changes. Can you merge the pr

**Comment by limzykenneth** [comment]
Looks good. Thanks!

**Pull request merged**